### ISSUE TITLE
Increasing/decreasing stroke moves Radius slider but value doesn't change

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#3663]
![stroke-radius](https://cloud.githubusercontent.com/assets/8442631/12798741/f120d33e-ca7f-11e5-822e-4d4cbeea2268.gif)

**Comment by iwehrman** [comment]
This is as designed. The max radius is based on the bounds of the shape and and changing the stroke width changes the bounds. Hence, the relative position of the slider changes when the stroke width changes.

**Comment by chadrolfs** [comment]
I'm not sure I agree from a user's perspective, especially if the radius has already been max'd out.  Also, forward thinking... how will we handle this if/when we implement per corner radii?

related to #3664

**Comment by chadrolfs** [comment]
Also, if you start toggling stroke visibility, should you really see other things move?

**Comment by iwehrman** [comment]
I agree that there is something funny here, especially w.r.t. to #3664. I also agree that toggling stroke visibility, by the current design, should cause the slider to update since it causes the bounds to update as well.
